core/src/test/java/com/gentics/mesh/core/graphql/GraphQLNodeLanguageSearchEnpointTest.java

```diff
@@ -0,0 +1,53 @@
+package com.gentics.mesh.core.graphql;
+
+
+import com.gentics.mesh.FieldUtil;
+import com.gentics.mesh.core.rest.node.NodeCreateRequest;
+import com.gentics.mesh.core.rest.schema.impl.SchemaReferenceImpl;
+import com.gentics.mesh.core.rest.user.NodeReference;
+import com.gentics.mesh.parameter.impl.NodeParametersImpl;
+import com.gentics.mesh.test.TestSize;
+import com.gentics.mesh.test.context.MeshTestSetting;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static com.gentics.mesh.test.ClientHelper.call;
+import static com.gentics.mesh.test.TestDataProvider.PROJECT_NAME;
+
+@MeshTestSetting(useElasticsearch = true, testSize = TestSize.FULL, startServer = true)
+@RunWith(Parameterized.class)
```

Comment:
I usually switch the order so that shorter annotations are on top. You must not change this.